std/concurrency.d

```diff
@@ -1967,9 +1967,11 @@ private
                 enum timedWait = false;
             }
 
+            static assert(Ops.length);
+
             bool onStandardMsg(ref Message msg)
             {
-                foreach (i, t; Ops)
+                static foreach (i, t; Ops)
```

Comment:
is this newly `static` foreach run with more iterations than a one at some point ? 
I ask because the two declarations that follow should create an error in this case.

Reply:
Ah... I see. I didn't have any case so far... But nevertheless, it would be wrong then.